### PR TITLE
Code cleanup for duplicate mock data of Chat & Template

### PR DESCRIPTION
**Pull request by DigneshGujarathi**

### CODE REVIEW

File: src/common/constants.ts [hunk 1]
```diff
@@ -61,6 +61,18 @@ export const CONNECTION_RECONNECT_ATTEMPTS = 5;
 
 export const MEDIA_MESSAGE_TYPES = ['IMAGE', 'AUDIO', 'VIDEO', 'DOCUMENT', 'STICKER'];
 
+export const getObject = (arr: any, data: any) => {
```
Comment (kurund) [inline src/common/constants.ts:64]: This does not belong to constants. Can you create common/utils.ts and move this function there.

File: src/containers/Chat/ChatConversations/ChatConversations.tsx [hunk 2]
```diff
@@ -167,35 +167,26 @@ export const ChatConversations: React.SFC<ChatConversationsProps> = (props) => {
...
-        className={styles.BackgroundWhite}
-        onClick={(e: any) => {
-          handleClick(e, 'saveSearch', 'update');
-        }}
-      >
-        Update
-      </Button>
-    </Tooltip>
-  ) : null;
+  const button = (toolTip: string, type: string, label: string) => {
```
Comment (kurund) [inline src/containers/Chat/ChatConversations/ChatConversations.tsx:170]:
```suggestion
  const buildButton = (toolTip: string, type: string, label: string) => {
```

File: src/containers/Chat/ChatConversations/ChatConversations.tsx [hunk 3]
```diff
@@ -167,35 +167,26 @@ export const ChatConversations: React.SFC<ChatConversationsProps> = (props) => {
...
-        className={styles.BackgroundWhite}
-        onClick={(e: any) => {
-          handleClick(e, 'saveSearch', 'new');
-        }}
-      >
-        Create new
-      </Button>
-    </Tooltip>
-  );
+  const btnUpdate = savedSearchCriteriaId ? button(toolTip, 'update', 'Update') : null;
```
Comment (kurund) [inline src/containers/Chat/ChatConversations/ChatConversations.tsx:187]:
```suggestion
  const btnUpdate = savedSearchCriteriaId ? buildButton(toolTip, 'update', 'Update') : null;
```

File: src/containers/Chat/ChatConversations/ChatConversations.tsx [hunk 4]
```diff
@@ -167,35 +167,26 @@ export const ChatConversations: React.SFC<ChatConversationsProps> = (props) => {
...
+
+  const btnCreate = button('Create a new collection', 'new', 'Create new');
```
Comment (kurund) [inline src/containers/Chat/ChatConversations/ChatConversations.tsx:189]:
```suggestion
  const btnCreate = buildButton('Create a new collection', 'new', 'Create new');
```

**Review by kurund — CHANGES_REQUESTED** [review]

File: src/containers/Chat/ChatConversations/ChatConversations.tsx (hunk 4 above)
Comment (DigneshGujarathi) [reply src/containers/Chat/ChatConversations/ChatConversations.tsx]: I updated the code, you can review it, please

**Review by kurund — APPROVED** [review]
looks good!

**Pull request merged**